src/components/Layout/PageWrapper/index.js

```diff
@@ -16,7 +15,6 @@ const PageWrapper = ({ children, location, pageHeader }) => {
   }
   return (
     <>
-      <SkipToMain location={location} />
```

Comment:
Why did this get removed and not replaced?

Reply:
It was forfeited instead for use of the `<main />` sectioning tab because the #mainContent anchor link is broken.